src/AtvService.php

```diff
@@ -457,17 +445,15 @@ public function getUserDocuments(string $sub, string $transaction_id = ''): arra
     if (!empty($transaction_id)) {
       $params['transaction_id'] = $transaction_id;
     }
-
+    $useApiKey = TRUE;
```

Comment:
Miks tää laitetaan tässä TRUEksi?

Täähän käytännössä estää käyttäjän dokumenttien hakemisen omaa tokenia käyttäen. Miks me haluttais sellasta tehdä?

Ton apikeyn ajatus on, että luokka voi asioida ATV:n kanssa joko käyttäjän omalla tokenilla tai yleisellä apikeyllä jos ei omaa tokenia ole saatavilla TAI on konffittu käyttää apikeytä.

Reply:
Aiemminkin koodi on käyttänyt API keytä.
 
Me ei pikaisella vilkaisulla tätä käytetä tätä käytetä muuhun kuin käyttäjälle näytettävän datan esittämiseen.

ATV:n API dokumentaatiossa /userdocuments (List users documents parts that doesn't contain sensitive information to easily see current applications and documents of a single user across services.) on dokumentoitu eri tavoin /documents (vain käyttäjä saa hakea) joten ennen kuin tätä muuttaa pitää varmistaa että muutos toimii ATV:n kanssa.